include/extractor/guidance/debug.hpp

```diff
@@ -0,0 +1,46 @@
+#ifndef OSRM_EXTRACTOR_GUIDANCE_DEBUG_HPP_
```

Comment:
Can we put all this debug logic in a central debug header (`util/debug.hpp`)? We don't need to be worried about a star topology with dependencies since this header should never be included in production code.